CHANGELOG.md

```diff
@@ -1,6 +1,47 @@
 
 # Change Log
 
+## 0.8.1 (Target May 30, 2023)
+
+BUG FIXES:
+
+* Explicitly setting `account_id` and `account_name` variables during AWS Account Setup Wizard [#430](https://github.com/noqdev/iambic/pull/430)
+* Create iambic docker user in the docker image creation [#435](https://github.com/noqdev/iambic/pull/435)
```

Comment:
I think this should be `Create iambic docker user before assigning file permissions"